assets/stylesheets/custom.css.scss

```diff
@@ -7,6 +7,16 @@ body {
   text-decoration: none;
 }
 
+.navbar-inverse .navbar-brand {
+  font-size: 1.69em;
+  font-weight: 500;
+}
+
+.navbar-inverse .navbar-collapse {
+  font-size: 1.42em;
+  font-weight: 300;
```

Comment:
Font sizes do not align with the default. I would take this out for now. We can worry about it later.